src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLQueryTypeImpl.java

```diff
@@ -593,8 +593,8 @@
 
       CallerWithEnvironment callWith = new CallerWithEnvironment(environment);
 
-      callWith.argument("fromPlace", request::setFromString);
-      callWith.argument("toPlace", request::setToString);
+      callWith.argument("fromPlace", (Consumer<String>) request::setFromString);
+      callWith.argument("toPlace",  (Consumer<String>) request::setToString);
```

Comment:
Was macht `(Consumer<String>)`?

Reply:
Dieses Consumer braucht man um in dem Java Lambda Expression(Das mit den :: und so) zu sagen welche der überladennen Methoden er hier verwenden soll. 

Ich hatte ja die Setter Methoden überladen, darum war die Änderung nötig.

Reply:
Und hier sagst du er soll die alte Funktion (`setToString(String to)`) verwenden statt deine neue?

Reply:
Danke für die Erklärung